constraintlayout/compose/src/main/java/androidx/constraintlayout/compose/ConstraintLayoutBaseScope.kt

```diff
@@ -384,6 +385,102 @@ abstract class ConstraintLayoutBaseScope {
         return HorizontalAnchor(id, 0, LayoutReferenceImpl(id))
     }
 
+
+    /**
+     * Wrap defines the type of chain
+     */
+    class Wrap private constructor(
+        internal val mode: Int) {
+        companion object {
+            val None = Wrap(androidx.constraintlayout.core.state.helpers.FlowReference.WRAP_NONE)
+            val WrapChain = Wrap(androidx.constraintlayout.core.state.helpers.FlowReference.WRAP_CHAIN)
+            val WrapAligned = Wrap(androidx.constraintlayout.core.state.helpers.FlowReference.WRAP_ALIGNED)
+            val WrapChainNew = Wrap(androidx.constraintlayout.core.state.helpers.FlowReference.WRAP_CHAIN_NEW)
+
+        }
+    }
+
+    /**
+     * Defines how objects align vertically within the chain
+     */
+    class VerticalAlign private constructor(
+        internal val mode: Int) {
+        companion object {
+            val Top =  VerticalAlign(androidx.constraintlayout.core.widgets.Flow.VERTICAL_ALIGN_TOP)
+            val Bottom =  VerticalAlign(androidx.constraintlayout.core.widgets.Flow.VERTICAL_ALIGN_BOTTOM)
+            val Center =  VerticalAlign(androidx.constraintlayout.core.widgets.Flow.VERTICAL_ALIGN_CENTER)
+            val Baseline =  VerticalAlign(androidx.constraintlayout.core.widgets.Flow.VERTICAL_ALIGN_BASELINE)
+        }
+    }
+
+    /**
+     * Defines how objects align horizontally in the chain
+     */
+    class HorizontalAlign private constructor(
+        internal val mode: Int) {
+        companion object {
+            val Start =  HorizontalAlign(androidx.constraintlayout.core.widgets.Flow.HORIZONTAL_ALIGN_START)
+            val End =  HorizontalAlign(androidx.constraintlayout.core.widgets.Flow.HORIZONTAL_ALIGN_END)
+            val Center =  HorizontalAlign(androidx.constraintlayout.core.widgets.Flow.HORIZONTAL_ALIGN_CENTER)
+         }
+    }
+
+    /**
+     * Defines how widgets are spaced in a chain
+     */
+    class FlowStyle private constructor(
+        internal val style: Int) {
+        companion object {
+            val Spread =  FlowStyle(0)
+            val SpreadInside =  FlowStyle(1)
+            val Packed =  FlowStyle(2)
+        }
+    }
+
+    /**
+     * This creates a flow helper
+     * Flow helpers allows a long sequence of Composable widgets to wrap onto
+     * multiple rows or columns.
+     */
+    fun createFlow(
```

Comment:
Javadoc for every parameter.

For example, it's not clear what `maxElement` means.